__tests__/tests.js

```diff
@@ -39,3 +40,85 @@ test("no console errors and warnings", async () => {
 
   expect(errors).toHaveLength(0);
 });
+
+test("every city is in the toggle", async () => {
+  const data = fs.readFileSync("data/cities-polygons.geojson");
+  const expectedCities = JSON.parse(data)["features"].map(
+    (entry) => entry["properties"]["Name"]
+  );
+  expectedCities.push("Select a city");
+
+  const browser = await puppeteer.launch();
+  const page = await browser.newPage();
+  await page.goto("http://localhost:8080");
+
+  const toggleValues = await page.evaluate(() => {
+    const select = document.querySelector("#city-choice");
+    const options = select.querySelectorAll("option");
+    const values = [];
+    for (let i = 0; i < options.length; i++) {
+      values.push(options[i].textContent.trim());
+    }
+    return values;
+  });
+  await browser.close();
+
+  toggleValues.sort();
+  expectedCities.sort();
+  expect(toggleValues).toEqual(expectedCities);
+});
+
+test("correctly load the city score card", async () => {
+  const data = fs.readFileSync("data/cities-polygons.geojson");
+  const anchorageEntries = JSON.parse(data)
+    ["features"].filter(
+      (entry) => entry["properties"]["Name"] === "Anchorage, AK"
+    )
```

Comment:
`filter` removes all elements from the array that don't return `bool` with this inline function